website/content/docs/k8s/connect/observability/metrics.mdx

```diff
@@ -129,17 +129,15 @@ ui:
     baseURL: http://prometheus-server
 ```
 
-## Deploying Prometheus and Grafana (_for demo and non-production use-cases only_)
+## Deploying Prometheus (_for demo and non-production use-cases only_)
 
-The Helm chart contains demo manifests for Prometheus and Grafana. They can be installed with Helm via `prometheus.enabled` and `grafana.enabled`. These manifests are based on the community manifests for Prometheus and Grafana.
-These are designed to allow quick bootstrapping for trial and demo use cases and not for production use-cases.
+The Helm chart contains demo manifests for Prometheus for viewing metrics within the Consul UI as was as through other external observability tools such as Grafana. It can be installed with Helm via `prometheus.enabled`. This manifest is based on the community manifest for Prometheus.
```

Comment:
```suggestion
The Helm chart contains demo manifests for deploying Prometheus. It can be installed with Helm via `prometheus.enabled`. This manifest is based on the community manifest for Prometheus.
```
I removed the sentence about viewing metrics in the UI since it seemed extraneous as this is mentioned in the previous section.